src/Routes/NavRoutes.js

```diff
@@ -0,0 +1,28 @@
+import { Routes, Route } from "react-router-dom";
+import { Home } from "../pages/Home/Home";
+import { Login } from "../pages/Login/Login";
+import { Playlists } from "../pages/Playlists/Playlists";
+import { Liked } from "../pages/Liked/Liked";
+import { WatchLater } from "../pages/WatchLater/WatchLater";
+import { History } from "../pages/History/History";
+import { Logout } from "../pages/Logout/Logout";
+import { Signup } from "../pages/Signup/Signup";
+import Mockman from "mockman-js";
```

Comment:
suggestion: create a index file in pages folder, and import all your pages from that file